app.py

```
@@ -401,16 +435,40 @@ def api_stop_simulation():
-    print("DragonCP Web UI starting...")
-    print("Access the application at: http://localhost:5000")
+    runtime_port = _get_runtime_port()
+    debug_mode = os.environ.get('FLASK_DEBUG', '0') == '1'
+
+    logger.info('DragonCP Web UI starting on port %s (debug=%s)', runtime_port, debug_mode)
+    logger.info('Access the application at: http://localhost:%s', runtime_port)
     
-    socketio.run(app, host='0.0.0.0', port=5000, debug=True, allow_unsafe_werkzeug=True)
+    socketio.run(
+        app,
+        host='0.0.0.0',
```

Ruff (0.15.4)

[error] Possible binding to all interfaces

(S104)